pkg/v1/providers/ytt/02_addons/management-packages/addons-manager/tanzu-addons-manager_overlay.lib.yaml

```diff
@@ -9,6 +9,6 @@ metadata:
   #@overlay/match missing_ok=True
   annotations:
     ext.packaging.carvel.dev/ytt-data-values-overlays: "tanzu-addons-manager-data-values"
-    tkg.tanzu.vmware.com/addon-type: "addons-management/tanzu-addons-manager"
+    tkg.tanzu.vmware.com/addon-type: "packages/management-package"
```

Comment:
Question: Do all the management packages share the same `addon-type`?

Reply:
@blc1996 yes, they use the same addon-type: "packages/management-package"